database/db_build.sql

```diff
@@ -0,0 +1,59 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users, careers, admins, universities, users_careers, universities_careers cascade;
+
+CREATE TABLE users(
+  user_id               SERIAL    PRIMARY KEY,
```

Comment:
I would leave all the primary keys as just `id`. The only tables with a different name for `id`s should be the relational tables. `user_careers` and university_careers